pandas/tests/io/parser/test_encoding.py

```diff
@@ -20,9 +20,9 @@
 import pandas._testing as tm
 
 skip_pyarrow = pytest.mark.usefixtures("pyarrow_skip")
+xfail_pyarrow = pytest.mark.usefixtures("pyarrow_xfail")
```

Comment:
How many tests are this? xfail kinda kills test runtime compared to skip

Reply:
It's like 10. The input data is tiny so it shouldn't matter. There is still a hanging test, so we still have ``skip_pyarrow`` in there.

Reply:
ok should be fine then